packages/core/src/connection/transactional-connection.ts

```diff
@@ -73,21 +74,32 @@ export class TransactionalConnection {
     getRepository<Entity extends ObjectLiteral>(
         ctx: RequestContext | undefined,
         target: ObjectType<Entity> | EntitySchema<Entity> | string,
+        replicationMode?: ReplicationMode,
     ): Repository<Entity>;
     getRepository<Entity extends ObjectLiteral>(
         ctxOrTarget: RequestContext | ObjectType<Entity> | EntitySchema<Entity> | string | undefined,
         maybeTarget?: ObjectType<Entity> | EntitySchema<Entity> | string,
+        replicationMode?: ReplicationMode,
```

Comment:
Here it's probably better to use an object instead of a strict arg

Reply:
For future extensibility?

Reply:
yes